.github/workflows/integration-test-helm-suite.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions: ["v1.22.17", "v1.27.2"]
+        kubernetes-versions: ["v1.23.17", "v1.28.0"]
```

Comment:
Let's revert all the changes that increase the min version, for two purposes:
1. We can backport it to 1.12 and avoid some merge conflicts
2. When it is closer to the 1.13 release then we can increase the min version for that release.

Reply:
agreed, reverted